web/modules/custom/dpl_example_content/content/menu_link_content/ff6db3ce-49e0-4107-b283-3403e22c5e6d.yml

```diff
@@ -0,0 +1,47 @@
+_meta:
+  version: '1.0'
+  entity_type: menu_link_content
+  uuid: ff6db3ce-49e0-4107-b283-3403e22c5e6d
+  bundle: menu_link_content
+  default_langcode: en
+default:
+  enabled:
+    -
+      value: true
+  title:
+    -
+      value: 'Det sker'
+  menu_name:
+    -
+      value: main
+  link:
+    -
+      uri: 'internal:/'
```

Comment:
Consider linking to the `/events` page.

It is a reasonable assumption that this menu item would actually take the user there.

Reply:
Good point, done 👍